frontends/mrxl/test/dot.mrxl

```diff
@@ -1,5 +1,5 @@
 input avec: int[4]
 input bvec: int[4]
-output dot: int
+output dot: int[4]
```

Comment:
Maybe this is a dumb question, but why would the output for a dot product be an array (instead of a scalar)?